ClientTests/TrackingAdsTests.swift

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+@testable import Firefox_Focus
+import WebKit
+
+class TrackingAdsTests: XCTestCase {
+    
+    let tpm = TrackingProtectionManager(isTrackingEnabled: {
+        Settings.getToggle(.trackingProtection)
+    })
```

Comment:
In test you can mock it by simply return true or false

Reply:
not applicable anymore, I dont need TrackingProtectionManager after I separated the tracking ads into its own class